linux/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "0.12",
```

Comment:
My main thinking here is that the manifest only _has_ to change if these binaries change, and that's quite rare. (Most changes are in the JS layer of the add-on.) By leaving this number untouched until needed, we avoid extra unpacking for add-on upgrade where the binaries don't actually change.

But yes, that means there's an extra step to update these files when we _do_ change the binaries. I'll file a bug to think more about it.

Yes, I am planning to bump the add-on to 0.12 after merging this, so that's why I used that number here. I filed a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1440699).